src/controllers/client.js

```diff
@@ -194,20 +194,22 @@ axiosInstance.interceptors.response.use(
   }
 );
 
-const echisAxiosInstance = axios.create({
-  baseURL: CHT.url,
+const echisAxiosInstance =()=>{
```

Comment:
we can use the default axios instance with different request config i.e auth, base url etc. There are utility functions you can use for this in https://github.com/moh-kenya/echis-2.0-integration/blob/main/src/utils/echis.js